tests/test_svg.py

```diff
@@ -130,273 +130,19 @@ def test_diagram_saves(self, tmp_svg: SVGDiagram) -> None:
-                "stroke": "#4A4A97",
-            },
-            ".LogicalArchitectureBlank g.Edge.ComponentExchange > text": {
-                "fill": "#4A4A97",
-            },
-            ".LogicalArchitectureBlank g.Edge.FIPAllocation > rect": {
-                "fill": "none",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Circle.FIPAllocation > circle": {
-                "fill": "#E08503",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Edge.FIPAllocation > path": {
-                "stroke": "#E08503",
-            },
-            ".LogicalArchitectureBlank g.Edge.FOPAllocation > rect": {
-                "fill": "none",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Circle.FOPAllocation > circle": {
-                "fill": "#095C2E",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Edge.FOPAllocation > path": {
-                "stroke": "#095C2E",
-            },
-        }
-
+    def test_base_css_styles(self, tmp_json: pathlib.Path) -> None:
         tree = etree.fromstring(
             SVGDiagram.from_json_path(tmp_json).to_string()
         )
-        style_ = tree.xpath(
-            "/x:svg/x:defs/x:style",
-            namespaces={"x": "http://www.w3.org/2000/svg"},
-        )[0]
-
-        stylesheet = cssutils.parseString(style_.text)
-        for rule in stylesheet:
-            for element, prop in COLORS_TO_CHECK.items():
-                if (
-                    rule.type == rule.STYLE_RULE
-                    and rule.selectorText == element
-                ):
-                    for key, val in prop.items():
-                        try:
-                            property_value = rule.style.getProperty(
-                                key
-                            ).propertyValue.value
-                        except AttributeError:
-                            # FIXME: rules are duplicated with different
-                            # values -> should be merged first
-                            print(f"Missing attribute {key}")
-                            continue
-                        if val == "none":
-                            assert property_value == "none"
-                        elif key in ["fill", "stroke"]:
-                            assert (
-                                property_value
-                                == cssutils.css.ColorValue(val).value
-                            )
-                        else:
-                            raise NotImplementedError
+        style_ = tree.get("style")
+        assert style_ == (
+            """shape-rendering: geometricPrecision; font-family:
+                'Segoe UI'; font-size: 8pt; cursor: pointer;""".replace(
+                "\n", ""
+            )
+            .replace("   ", "")
+            .strip()
+        )
```

Comment:
I don't think it makes sense to check for the exact string here, because the contents are static and can only change by someone deliberately doing so – in which case that someone has to change it twice for no good reason. A more useful test here would be "is something there", i.e. implicit truthiness, to check that it wasn't accidentally deleted completely:

```suggestion
        style_ = tree.get("style")
        assert style_
```